apps/avatax/src/pages/api/webhooks/order-confirmed.ts

```diff
@@ -58,20 +51,20 @@ export default wrapWithLoggerContext(
           ctx.authData,
         );
 
-        // todo: figure out what fields are needed and add validation
-        if (!payload.order) {
-          return webhookResponse.error(new Error("Insufficient order data"));
-        }
+        const payloadVerificationResult = verifyOrderConfirmedPayload(payload);
 
-        if (payload.order.status === OrderStatus.Fulfilled) {
-          return webhookResponse.error(
-            new Error("Skipping fulfilled order to prevent duplication"),
-          );
+        if (payloadVerificationResult.isErr()) {
```

Comment:
You mentioned that if shipping is missing, billing will be a fallback. We do a defensive check here. But if this happens, we should be aware of that (contract was broken). So we should notify Sentry about this